crates/mem-pool/src/fee/types.rs

```diff
@@ -67,14 +75,20 @@ impl PartialOrd for FeeItem {
     }
 }
 
+#[derive(PartialEq, Eq, Clone, Copy, Debug, Hash)]
+pub enum FeeItemSender {
+    AccountId(u32),
+    PendingCreate(H256), // hash
```

Comment:
The hash here is never read? Is it for distinguishing different senders in the `fetched_senders` map?

Reply:
Yes, for `fetched_senders` map.